pkg/extraconfig/extraconfig.go

```diff
@@ -78,7 +78,7 @@ func GetExtraConfig() (*ExtraConfig, error) {
 		case "awsCredentialFile":
 			extraConfig.AwsCredentialFile = val
 		default:
-			os.Setenv(key, val)
+			_ = os.Setenv(key, val)
```

Comment:
i think lint errors have started to come up after v2 bump, it makes sense for most of these, maybe we can check and return error if we find anywhere useful